lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -445,11 +445,21 @@ def main():
                 checksum_url = checksum
                 # download checksum file to checksum_tmpsrc
                 checksum_tmpsrc, checksum_info = url_get(module, checksum_url, dest, use_proxy, last_mod_time, force, timeout, headers, tmp_dest)
-                lines = [line.rstrip('\n') for line in open(checksum_tmpsrc)]
-                os.remove(checksum_tmpsrc)
+                with open(checksum_tmpsrc) as f:
+                    lines = [line.rstrip('\n') for line in f]
```

Comment:
Did the os.remove() get dropped unintentionally?

Reply:
Yes it did. It should have been moved below the context manager statement.